src/enc_jpeg2000.c

```diff
@@ -56,14 +56,14 @@
  * 1 = try increasing number of guard bits otherwise, no additional options
  * @param[in] jpclen  - Number of bytes allocated for new JPEG2000 code stream in outjpc.
  * @param[in] outjpc - Output encoded JPEG2000 code stream
- * @return - > 0 = Length in bytes of encoded JPEG2000 code stream
+ * @return 0 = Length in bytes of encoded JPEG2000 code stream
```

Comment:
You can remove the = as well, so just:

`@return 0  Length in bytes of encoded JPEG2000 code stream
`

But is this correct? If it returns 0, how is that the length of the stream?

Reply:
Will remove that "=".
The return code just show that the code works good. This will be rewords.